daemon/volumes_unix.go

```diff
@@ -92,6 +92,11 @@ func parseBindMount(spec, volumeDriver string) (*mountPoint, error) {
 	}
 
 	if len(source) == 0 {
+		//validate the name of named volume
+		if strings.HasPrefix(name, "./") || strings.HasPrefix(name, "../") {
```

Comment:
`.` and `..` are problems as well... and I suspect anything that looks like a path would cause some issues.